.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,25 @@
+name: Chromatic
+
+on:
+  push:
```

Comment:
@alexgrozav do we want to change this to run once every night instead ?

Reply:
updated to run once at midnight. We have a limited number of snapshots available on chromatic. if we run this job on every push, we'll likely run out of quota in a couple of days.